cmd/cmd.go

```diff
@@ -879,6 +879,24 @@ func NewCLI() *cobra.Command {
 		Args:    cobra.ExactArgs(0),
 		RunE:    RunServer,
 	}
+	serveCmd.SetUsageTemplate(serveCmd.UsageTemplate() + `
+Environment Variables:
+
+	OLLAMA_HOST
+		The host:port to bind to (default "localhost:11434")
+
+		Examples:
+			"localhost:11434"
+	OLLAMA_ORIGINS
+		A comma separated list of allowed origins. If unset, the
+		default behavior is to allow same origin requests, only.
+
+		Examples:
+			"localhost:11434"
+			"example.com,api.example.com"
+	OLLAMA_MODELS
+		The path to the models directory (default is "~/.ollama/models")
```

Comment:
I'd prefer to avoid dynamic CLI docs if possible so they remain consistent for all users. If we do decide to introduce dynamic components to docs, I think we can try that in another PR.